nautobot/dcim/choices.py

```diff
@@ -105,8 +105,9 @@ class RackDimensionUnitChoices(ChoiceSet):
 class RackElevationDetailRenderChoices(ChoiceSet):
     RENDER_JSON = "json"
     RENDER_SVG = "svg"
+    RENDER_CSV = "csv"
 
-    CHOICES = ((RENDER_JSON, "json"), (RENDER_SVG, "svg"))
+    CHOICES = ((RENDER_JSON, "json"), (RENDER_SVG, "svg"), (RENDER_CSV, "csv"))
```

Comment:
Nit: Since the second element in each tuple is the "human-facing" label, I wonder if this should really be:

```suggestion
    CHOICES = ((RENDER_JSON, "JSON"), (RENDER_SVG, "SVG"), (RENDER_CSV, "CSV"))
```